src/app/privacy-policy/page.tsx

```diff
@@ -19,13 +24,26 @@ const Page = () => {
 
             {section.title === 'SUMMARY OF KEY POINTS' && (
               <>
-                {accordionData.map((item, accordionIndex) => (
-                  <Accordion
-                    key={accordionIndex}
-                    title={item.title}
-                    content={item.content}
-                  />
-                ))}
+                {accordionData.map((item, accordionIndex) => {
+                  const email = 'samahan@addu.edu.ph';
+
+                  const updatedContent = item.content.replace(
+                    email,
+                    `<a href="mailto:${email}" class="font-bold">${email}</a>`
+                  );
+
+                  return (
+                    <Accordion
+                      key={accordionIndex}
+                      title={item.title}
+                      content={
+                        <span
+                          dangerouslySetInnerHTML={{ __html: updatedContent }}
+                        />
+                      }
```

Comment:
para saan itu?

Reply:
every single email address is a mailto :D

Reply:
oks oks